pymc3/sampling.py

```diff
@@ -823,7 +862,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     @staticmethod
     def _run_slave(c, stepper_dumps, slave_end):
-        """Started on a separate process to perform stepping of a chain.
+        """Start a separate process to perform stepping of a chain.
```

Comment:
No, this is wrong.
Should be: `This method is started on a separate process...`